### PR TITLE
update for @rthallisey suggestions from PR#25

### PR DESCRIPTION
**Pull request by johnkim76**

### CODE REVIEW

File: docs/developers.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ In order to create an APB, you will need to start with a skeleton APB directory
...
 
-Running the `apb init` will create a APB directory structure shown below:
+Run the `apb init` command
 ```bash
 $ apb init my-apb
-
-$ tree my-apb
+```
+The following APB directory structure will be created:
+```bash
```
Comment (rthallisey) [inline docs/developers.md]: I saw in the design doc we have a sub field that has ```##### Directory Structure```.   Maybe we can do something like that?
https://github.com/fusor/ansible-playbook-bundle/blob/master/docs/design.md#directory-structure

**Review by rthallisey — CHANGES_REQUESTED** [review]

File: docs/developers.md (hunk 1 above)
Comment (johnkim76) [reply docs/developers.md]: when the `apb init my-apb` is ran, the folder structure is as shown.  It's also documented in the [getting started ](https://github.com/fusor/ansible-playbook-bundle/blob/master/docs/getting_started.md#creating-your-first-apb)

Comment (johnkim76) [reply docs/developers.md]: if that section is not clear, should I just mention to run the `apb init` coommand and remove showing the directory structure?

Comment (rthallisey) [reply docs/developers.md]: I think the section is clear. I like the addition of the directory structure and I think it belongs. My suggestion is just the addition of the title to follow the same pattern from the design doc. IMO with or without it works for me.

Comment (johnkim76) [reply docs/developers.md]: Oh, I see, yes, I will add the title

**Review by rthallisey — APPROVED** [review]

File: docs/developers.md [hunk 2]
```diff
@@ -93,7 +96,6 @@ And a simple `deprovision.yml` may look like this.
 We will also need to create the Ansible roles as specified in the actions. The below directory structure shows what it can look like:
 
 ```bash
-$ tree my-apb
 my-apb/
```
Comment (rthallisey) [inline docs/developers.md:100]: same as above:. ```### Directory Structure```

Comment (johnkim76) [reply docs/developers.md:100]: thanks, I've made the updates

**Comment by rthallisey** [comment]
👍

**Pull request merged**